src/lib/report-generator/helpers.js

```diff
@@ -118,6 +118,10 @@ const isAssertionPassed = (status) => {
   return status === 'SUCCESS'
 }
 
+const isAssertionSkipped = (input) => {
+  return input.status === 'SKIPPED' && input.skipped
```

Comment:
I think if you set the status to 'SKIPPED', we don't need to use skipped flag

Reply:
fixed